build.gradle.kts

```diff
@@ -63,8 +68,8 @@ if (idea.project != null) { // may be null during script compilation
         project {
             settings {
                 taskTriggers {
-                    afterSync(":sdk-codegen:generateSdks")
-                    afterSync(":jetbrains-core:generateTelemetry")
+                    afterSync(":plugin-core:sdk-codegen:generateSdks")
+                    afterSync(":plugin-toolkit:jetbrains-core:generateTelemetry")
```

Comment:
Should this also be a part of plugin-core?

Reply:
that is more involved so will happen later